src/Discovery/Discovery.tsx

```diff
@@ -316,13 +316,51 @@ const Discovery: React.FunctionComponent<DiscoveryBetaProps> = (props: Discovery
     setSearchFilteredResources(results);
   };
 
-  const handleAccessLevelChange = (ev) => {
-    const value = ev.target.value as AccessLevel;
-    setAccessLevel(value);
+  const handleExportToWorkspaceClick = async () => {
+    const manifestFieldName = config.features.exportToWorkspaceBETA.manifestFieldName;
+    if (!manifestFieldName) {
+      throw new Error('Missing required configuration field `config.features.exportToWorkspaceBETA.manifestFieldName`');
+    }
+    // combine manifests from all selected studies
+    const manifest = [];
+    selectedResources.forEach((study) => {
+      if (study[manifestFieldName]) {
+        manifest.push(...study[manifestFieldName]);
+      }
+    });
+    // post selected resources to manifestservice
+    const res = await fetchWithCreds({
+      path: `${manifestServiceApiPath}`,
+      body: JSON.stringify(manifest),
+      method: 'POST',
+    });
+    if (res.status !== 200) {
+      throw new Error(`Encountered error while exporting to Workspace: ${JSON.stringify(res)}`);
+    }
+    // redirect to Workspaces page
+    props.history.push('/workspace');
+  };
+
+  const handleDownloadManifestClick = () => {
+    const manifestFieldName = config.features.exportToWorkspaceBETA.manifestFieldName;
+    if (!manifestFieldName) {
+      throw new Error('Missing required configuration field `config.features.exportToWorkspaceBETA.manifestFieldName`');
+    }
+    // combine manifests from all selected studies
+    const manifest = [];
+    selectedResources.forEach((study) => {
+      if (study[manifestFieldName]) {
+        manifest.push(...study[manifestFieldName]);
+      }
+    });
+    // download the manifest
+    const MANIFEST_FILENAME = 'manifest.json';
+    const blob = new Blob([JSON.stringify(manifest, null, 2)], { type: 'text/json' });
+    FileSaver.saveAs(blob, MANIFEST_FILENAME);
   };
 
   const visibleResources = filterByTags(
-    filterByAccessLevel(searchFilteredResources, accessLevel, accessibleFieldName),
+    searchFilteredResources,
```

Comment:
Why this change?

Reply:
I switched up the way the table filters by access -- now it leverages builtin filtering in the antd table. So we don't need the `filterByAccessLevel` function anymore so I removed it!